### PR TITLE
Bump cri-o versions

### PR DESCRIPTION
**Pull request by rhrazdil**
Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

### CODE REVIEW

File: cluster-provision/k8s/1.22-ipv6/provision.sh [hunk 1]
```diff
@@ -94,7 +94,7 @@ export PATH=$ISTIO_BIN_DIR:$PATH
 # generate Istio manifests for pre-pulling images
 istioctl manifest generate --set profile=demo --set components.cni.enabled=true | tee /tmp/istio-deployment.yaml
 
-export CRIO_VERSION=1.20
+export CRIO_VERSION=1.21
```
Comment (xpivarc) [inline cluster-provision/k8s/1.22-ipv6/provision.sh:97]: It is recommended to match crio with kubernetes version.

Comment (rhrazdil) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: Yes, but we don't heve 1.22 mirrored yet. I could do this only for 1.21 for now and post another PR for 1.22 when It's done

Comment (xpivarc) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: I see. I thought we don't run provisioning that often.

Comment (xpivarc) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: Keep in mind we already have 1.23 :)

Comment (xpivarc) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: Maybe we can switch to 1.22 now as the mirror pr was merged. WDYT?

Comment (dhiller) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: qq: who is handling the mirroring?

Comment (xpivarc) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: fyi https://github.com/kubevirt/project-infra/pull/1878

Comment (rhrazdil) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: > Maybe we can switch to 1.22 now as the mirror pr was merged. WDYT?

we had to wait for the sync to be triggered, which is at Sunday night.
Unfortunately, it failed. https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirtci-mirror-crio-repository-weekly/1487945070802898944

@dhiller This is the periodic job for syncing the repos https://github.com/kubevirt/project-infra/blob/2ffb5944e61b4b84068fc54613f175d7fd2f4d12/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml#L2

Comment (dhiller) [reply cluster-provision/k8s/1.22-ipv6/provision.sh:97]: > > Maybe we can switch to 1.22 now as the mirror pr was merged. WDYT?
> 
> we had to wait for the sync to be triggered, which is at Sunday night. Unfortunately, it failed. https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirtci-mirror-crio-repository-weekly/1487945070802898944
>

You can ping me, I can trigger the job manually if needed.

Comment (rhrazdil) [reply cluster-provision/k8s/1.22-ipv6/provision.sh]: Thank you @dhiller, it's already synced

**Comment by rhrazdil** [comment]
There seems to be an actual issue with crio-1.23, on `node02` There are failing pods with 
`Failed to create pod sandbox: rpc error: code = Unavailable desc = error reading from server: read unix @->/var/run/crio/crio.sock: read: connection reset by peer` error.

We'll need to investigate further.
For now, I'm pinning cri-o 1.22 on k8s-1.23

**Comment by xpivarc** [comment]
> There seems to be an actual issue with crio-1.23, on `node02` There are failing pods with `Failed to create pod sandbox: rpc error: code = Unavailable desc = error reading from server: read unix @->/var/run/crio/crio.sock: read: connection reset by peer` error.
> 
> We'll need to investigate further. For now, I'm pinning cri-o 1.22 on k8s-1.23
/lgtm
Thanks for having look!
Lets get this one in meanwhile

**Comment by rhrazdil** [comment]
@dhiller Could you review please?

**Review by dhiller — APPROVED** [review]
/approve

Sorry for the delay!

**Pull request merged**